pom.xml

```diff
@@ -98,6 +98,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
https://github.com/societe-generale/arch-unit-maven-plugin#how-to-use-archunit-maven-plugin-

You still need to configure some tests.